adaptors/commcare.md

```diff
@@ -1,219 +1,204 @@
+   or the desintation for the data you want to forward
+6. If you have [webhook](https://docs.openfn.org/documentation/webhook-security)
    set up on OpenFn, add the authentication type, the username and password here
 7. You can test the connection, then save it
 
 ![Connection](/img/commecare_connection_settings.png)
 
-### Forwarding Individual Forms
+#### Forwarding Data from CommCare
+
+There are two approaches to forwarding data from CommCare to an external
+platform.
+
+- Forwarding individual forms
+- Forwarding specific case types
+
+**Forwarding Individual Forms**
+
+Forwarding individual forms is useful when you want to forward a specific form.
+Here's how you can configure CommCare to forward a form to OpenFn.
 
 1. Click over to "Data Forwarding".
-2. Under "Forward Forms", click "+ Add a service to forward to"
+2. Go to the "Forward Forms" section and click "Add a service to forward to"
 3. Select the connection to forward the forms to set up following the steps
-   above
-4. Name it
+   above or add a new connection
+4. Enter the name of the forwarder e.g. _Forward Visit Forms to OpenFn_
 5. Select "POST" HTTP Request Method
 6. Choose "JSON" as Payload Format
 7. Exclude any (eg. test) users - forms submitted by them won't be forwarded
-8. "XMLNSes of forms to include" lets you select which form(s) to forward by
-   adding the XMLNSes of the required forms. Follow
+8. if you want to forward all your forms, leave the "XMLNSes of forms to
+   include" box empty or enter the XLMN of the form(s) to be forwarded and
+   separate them with commas, spaces or newlines. Follow
    [this CommCare guide](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143979045/Finding+a+Form+s+XMLNS)
-   to find the XMLNS of any form. To add multiple, separate them with commas,
-   spaces or newlines. Leave empty to forward all forms.
-9. Hit "Start Forwarding" to save and activate
+   to find the XMLNS of any form.
+9. Click "Start Forwarding" to save and activate
 
 ![Forms](/img/commecare_forward_forms.png)
 
-### Forwarding Specific Case Types
+**Forwarding Specific Case Types**
+
+In some cases, you may want to forward a specific case type rather than all
+forms. Here's how you can configure CommCare to forward a case type to OpenFn.
 
-1. Go to "Data Forwarding".
-2. lick "+ Add a service to forward to" under "Forward Cases"
+1. Go to `Data Forwarding`.
+2. Navigate to `Forward Cases` and click `Add a service to forward to`
 3. Select the connection to forward the cases to
 4. Name the forwarder
-5. Select "POST" HTTP Request Method
-6. Choose "JSON" as Payload Format
-7. Select which case type(s) you want to forward, for example "patient"
+5. Select `POST` HTTP Request Method
+6. Choose `JSON` as Payload Format
+7. Select which case type(s) you want to forward, for example `patient`
 8. Exclude any (eg. test) users
 
 ![Cases](/img/commcare_forward_cases.png)
 
-## Data Forwarding and OpenFn Workflow Design
```

Comment:
@christad92 i think some of these information might be useful to someone. I will suggest we turn them into a community post and it's okay if we can have a banner saying this is outdated information.

Reply:
Added back for now until we figure out where to put this info